reservations/flightreservation/flightreservation.h

```diff
@@ -41,6 +41,8 @@ class FlightReservation : public AbstractReservationPlugin
     Q_SIGNALS:
         void extractedData();
         void addedToDatabase();
+        Q_SCRIPTABLE void update();
+        Q_SCRIPTABLE addFlightCard();
```

Comment:
With this, the plasmoid backend will connect to: org.kde.kdenow /Flight 's addFlightCard